### PR TITLE
Rescheduler Doc changes

### PR DESCRIPTION
**Pull request by ravisantoshgudimetla**
Rescheduler now works with only DaemonSet pods and it knows priorities.

### CODE REVIEW

File: content/en/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md [hunk 1]
```diff
@@ -55,6 +55,10 @@ To be considered critical, the pod has to run in the `kube-system` namespace (co
 
 The first one marks a pod a critical. The second one is required by Rescheduler algorithm.
 
+A pod could also be considered critical, if its priority is greater than or equal to system-critical-priority.
+
+
```
Comment (bsalamat) [inline content/en/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods.md]: nit: remove two of the blank lines?

**Comment by ravisantoshgudimetla** [comment]
Done PTAL.

**Comment by ravisantoshgudimetla** [comment]
Thanks @bsalamat for the review

**Pull request merged**